pallets/loans/src/functions.rs

```diff
@@ -631,21 +631,13 @@ impl<T: Config> Pallet<T> {
 			Error::<T>::InvalidWriteOffGroup
 		);
 
-		// ensure we have not exceeded the max number of write off groups
```

Comment:
Why can we delete the `ensure!` here?

Reply:
Because now, the own `BoundedVec` checks if it exceeds the limits on its `try_push()` method.

Reply:
Thank you!